certbot-apache/certbot_apache/_internal/parser.py

```diff
@@ -346,7 +347,8 @@ def filter_args_num(self, matches: str, args: int) -> List[str]:
 
         return filtered
 
-    def add_dir_to_ifmodssl(self, aug_conf_path: str, directive: str, args: List[str]) -> None:
+    def add_dir_to_ifmodssl(self, aug_conf_path: str, directive: str,
+                            args: Union[str, Sequence[str]]) -> None:
```

Comment:
Why `Union`? Every call site I've found seems to be passing in a list for `args`.

Reply:
I was building on the docstring description here that was stating that args may be a string. But indeed it is never used. Given this is an internal module, I think we are safe to change the documentation and the typing appropriately, and I did that to accept only a `Sequence[str]`.